app/src/main/java/com/google/droidjump/GameView.java

```diff
@@ -72,20 +88,27 @@ public void run() {
     private void receiveLevelDetails() {
         // TODO: Serialize current level data and put it in some container
         levelTimePoints = 200;
-        levelSpeed = 50;
+        levelSpeed = 20;
```

Comment:
I can revert it (when I was testing 50 was just too fast), there is no need in constant for this variable, because this value will be received from level data.

Reply:
okay. you can leave it at 50 then :)

Reply:
Done